.github/workflows/check-aliyun-maven.yml

```
@@ -0,0 +1,165 @@
+              }
+
+              core.setOutput('dependencies', JSON.stringify(dependencies));
+              return dependencies.length;
+            } catch (error) {
+              core.setFailed(`Failed to parse dependencies: ${error}`);
+            }
+
+      - name: Check Aliyun Maven availability
+        if: steps.parse-deps.outputs.dependencies != '[]'
+        id: check-aliyun
+        uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)